internal/provider/organization_resource.go

```diff
@@ -133,6 +213,26 @@ func (r *OrganizationResource) Read(ctx context.Context, req resource.ReadReques
 		}
 	}
 
+	if !data.GroupSync.IsNull() {
+		_, err := r.Client.GroupIDPSyncSettings(ctx, data.ID.ValueUUID().String())
+		if err != nil {
+			resp.Diagnostics.AddError("Client Error", fmt.Sprintf("unable to get organization group sync settings, got error: %s", err))
+			return
+		}
+
+		// data.GroupSync = ???
```

Comment:
I don't recall if you can write to a config block (blocks can't be computed) like this. I suspect you can't.

If you can, I assume it would just be to error if the post-apply plan isn't empty.
For that you'd need to use `types.ObjectValueFrom` and provide it the type of the object using a `map[string]attr.Type`. I've got an example on the template resource & data source.

Reply:
I think I figured something out that works 😄